internal/mockexporter/metrics.file

```diff
@@ -0,0 +1,10 @@
+# HELP http_request_duration_seconds request duration histogram
```

Comment:
is this file needed? I guess we will need a larger number of metrics when implement the automated test

Reply:
I just copied a random example for the net for this. My idea was to keep using the metrics download functionality to feed a huge file from a gist/pastebin without polluting the repo with it.

Reply:
Update: I checked on a local k8s cluster and it seems to work well :)

Reply:
ok so this file could be removed right?

Reply:
Removed in https://github.com/newrelic/nri-prometheus/pull/109/commits/f6fb74cd52f8282f660e44792ed4872c50ca00e1